src/main/scala/nodes/learning/GaussianMixtureModel.scala

```diff
@@ -16,11 +17,16 @@ import workflow.{Transformer, Estimator}
  * @param variances Cluster variances (diagonal)
  * @param weights Cluster weights.
  */
-class GaussianMixtureModel(
-    val means: DenseMatrix[Double],
-    val variances: DenseMatrix[Double],
-    val weights: DenseVector[Double])
-  extends Transformer[DenseVector[Double],DenseVector[Double]] with Logging {
+case class GaussianMixtureModel(
+    means: DenseMatrix[Double],
+    variances: DenseMatrix[Double],
+    weights: DenseVector[Double],
+    weightThreshold: Double = 1e-4)
+  extends Transformer[DenseVector[Double],DenseVector[Double]] {
+
+  private val gmmMeans = means.t
```

Comment:
Hmm okay so should I leave the convention as it is for now (each gaussian mean in `means` and gaussian var in `vars` is a column) and then just add documentation declaring the convention?